mm/map/vm_region.c

```diff
@@ -128,8 +128,10 @@ FAR void *vm_map_region(uintptr_t paddr, size_t size)
   return (void *)((uintptr_t)vaddr + (MM_PGMASK & paddr));
 
 error:
-  if (i)   /* undo alway mapped pages */
+  if (i)
     {
+      /* undo alway mapped pages */
```

Comment:
Undo

Reply:
That produces an error from the format check `mm/map/vm_region.c:131:7: error: If statement followed by garbage`

Reply:
Maybe you can put it after up_shmdt(); or maybe we need to modify nxstyle.c to fix this false positive

Reply:
Ah, you simply meant to capitalize the comment :)

I'll do that of course

Reply:
Yes, sorry for confusion:).